### PR TITLE
Fix typos in TerminalCommands

### PR DESCRIPTION
**Pull request by FireFingers21**

### CODE REVIEW

File: TerminalCommands.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ defaults write com.knollsoft.Rectangle todo -int 1
 
 ## Only allow drag-to-snap when modifier keys are pressed
 
-Modifier key values can be ORed together.
```
Comment (rxhanson) [inline TerminalCommands.md:106]: "ORed" is the correct term here. We can replace it with something like "Modifier key values can be combined by OR operation"

**Comment by rxhanson** [comment]
Thanks for contributing! One comment inline.

**Comment by FireFingers21** [comment]
Gotcha, I'm not sure if this is what you meant, but modifer keys can be combined if you add their sums together, so I've changed the wording to reflect that

**Comment by FireFingers21** [comment]
I've tested several different combinations of modifier keys and adding the sum to combine them does actually work. The same applies when creating macOS's built-in "Hot Corners" from Terminal, which I have also done and re-verified.

**Comment by rxhanson** [comment]
Ah, you're right! There's no bit overlap, so it's the same as addition in this case.

**Pull request merged**